client/src/components/Game/Board.js

```diff
@@ -48,24 +49,68 @@ const useStyles = makeStyles((theme) => ({
   prompt: {
     display: "flex",
   },
+  promptContent: props => ({
+    display: "block",
+    color: props.promptColor,
+    "margin-left": "auto",
+    "margin-right": "auto",
+    '&:first-letter': {
+      "text-transform": "capitalize",
+    },
+  }),
+  timer: props => ({
+    color: props.timerColor,
+  }),
   paper: {
     padding: "5px",
     margin: "10px"
   }
 }));
 
-const getRole = (player, gameState) => {
-  const {blueSpy, redSpy, blueGuessers, redGuessers} = gameState;
-  if (player === blueSpy) return "blue spy";
-  if (player === redSpy) return "red spy";
-  if (blueGuessers.includes(player)) return "blue guesser";
-  if (redGuessers.includes(player)) return "red guesser";
-}
-
 const isSpy = role => role === "blue spy" || role === "red spy";
+const isSpectator = role => role === "spectator"
+
+const Prompt = ({ state, timer, getRole }) => {
+  let prompt;
+  const { gameState } = state;
+  const { turn, hint } = gameState;
+  const [team, teamRole] = getRole.split(" ");
+  const timerColor = timer < 10 ? 'red' : 'black';
+  const classes = useStyles({promptColor: team, timerColor: timerColor});
 
-const Card = ({status, color, word, onClick}) => {
-  const classes = useStyles({status: status, color: color});
+  if (team !== turn) {
+    prompt = `${turn} team's turn`;
+  }
+  else {
+    if (hint === undefined) {
+      if (teamRole === 'spy')
+        prompt = 'Your turn';
+      else
+        prompt = "Waiting for your spymaster's hint";
+    }
+    else {
+      if (teamRole === 'spy')
+        prompt = 'Waiting for your guessers';
+      else
+        prompt = 'Your turn';
+    }
+  }
```

Comment:
```
switch (true) {
case team !== turn:
...
case hint === undefined:
...
}
```
using `switch (true)` and iterating cases allows you to avoid nested `if/else`